### ISSUE TITLE
In scheme support for concepts

### ISSUE THREAD

**Issue by leskneebone** [Kurrawong/vocedit#12]
Under Concept > Concept scheme relationships > in scheme, the add URI / label options are there. But for this property, it should be easy to select the active concept scheme without adding manually. 

In theory a concept could be in multiple concept schemes. But most of the time it will be in one. In scheme setting could be set as a default on creating a concept scheme, with an override option for concept editing.

**Comment by edmondchuc** [comment]
Yes, I agree. New concepts created should automatically have the relationship to the concept scheme created.
